--- libsrc\occ\Partition_Inter3d.cxx	Wed Mar 12 23:39:54 2014
+++ libsrc\occ\Partition_Inter3d_new.cxx	Wed Mar 12 23:58:31 2014
@@ -243,8 +243,8 @@
       Standard_Integer i, nbExt = anExtPS.NbExt();
       Extrema_POnSurf aPOnSurf;
       for (i = 1; i <= nbExt; ++i )
-	if (anExtPS.Value( i ) <= TolE)               // V6.3
-	  // if (anExtPS.SquareDistance( i ) <= TolE)   // V6.5
+	//if (anExtPS.Value( i ) <= TolE)               // V6.3
+	if (anExtPS.SquareDistance( i ) <= TolE)   // V6.5
 	  {
           aPOnSurf = anExtPS.Point( i );
           break;
